cmd/kube-apiserver/app/server.go

```diff
@@ -445,7 +445,9 @@ func BuildGenericConfig(s *options.ServerRunOptions, proxyTransport *http.Transp
 		return nil, nil, nil, nil, nil, fmt.Errorf("invalid authentication config: %v", err)
 	}
 
-	genericConfig.Authorizer, genericConfig.RuleResolver, err = BuildAuthorizer(s, sharedInformers)
+	genericConfig.Authorizer, genericConfig.RuleResolver, err =
```

Comment:
ugly style. Better define local `dialer := buildWebhookDialer(serviceResolver, proxyTransport)` and keep it on one line.